src/EventTeachingService.cpp

```diff
@@ -22,15 +22,17 @@ void EventTeachingService::enableLearn()
   bLearn = true;
   //DEBUG_SERIAL << F("> set learn mode ok") << endl;
   // set bit 5 in parameter 8
-  bitSet(controller->_mparams[8], 5);
+  controller->_mparams[PAR_FLAGS] |= PF_LRN;
+ // bitSet(controller->_mparams[8], 5);
 }
 
 void EventTeachingService::inhibitLearn() 
 {
   bLearn = false;
   //DEBUG_SERIAL << F("> learn mode off") << endl;
   // clear bit 5 in parameter 8
-  bitClear(controller->_mparams[8], 5);
+  controller->_mparams[PAR_FLAGS] &= ~PF_LRN;
```

Comment:
Good to see the constants used instead of numbers.

Reply:
Gives consistency with flag handling elsewhere and clearer